src/main/webapp/components/search-forms/editor.tsx

```diff
@@ -112,8 +143,18 @@ export const QueryEditor = (props: EditorProps) => {
       </Typography>
       <Divider />
       <Box width="100%" display="flex" flexDirection="row" height="100%">
-        <Box style={{ height: `calc(100% - 60px)`, width: 500 }}>
-          <QueryBuilder query={query} onChange={props.onChange} />
+        <Box style={{ height: `calc(100% - 134px)`, width: 500 }}>
```

Comment:
dunno if there's a better way to do this, I just adjusted the pixels until the vertical scrollbar disappeared from the editor

Reply:
Does this look okay on both 1080p and 4k?

Reply:
Yep :) Have to zoom out a bit to get the scroll bars to disappear in 1080p